scripts/validate_map.py

```diff
@@ -61,7 +61,7 @@ def get_codes_to_remove(codes: List) -> List:
 
 
 def check_code_name_empty(
-    df: pd.DataFrame, syn: Synapse, config: Dict, cohort: str, release: bool
+    df: pd.DataFrame, syn: Synapse, config: Dict, cohort: str, release: str
```

Comment:
Your description says you're accepting true false, but this is a str type hint.

Can you clarify?

Reply:
Also changes made here but accidentally committed to develop: https://github.com/Sage-Bionetworks/GENIE-Sponsored-Projects/commit/3f52f54fe4f43250a9b62401bc3de958e42b5a7f